### ISSUE TITLE
Panel should close when last containing connection/tab closes

### ISSUE THREAD

**Issue by CrazyOrange** [mRemoteNG/mRemoteNG#1684]
When the last connection/tab in panel closed the parent panel stays open. Imho it should close or there should be an option for it.
Keeping open empty panels messes up the tab view.

I'm using version 1.77.1.27654 on windows 10.

**Comment by RickMay57** [comment]
That was a behavior that preexisted the current version, I would like to add my voice to this suggestion. It seems counter intuitive to a clean front-end

**Comment by GeriZ** [comment]
Yes, it would be better to close the panel after the last connection of this panel was closed!!

**Comment by farosch** [comment]
Duplicate of #1657
